torch/serialization.py

```diff
@@ -419,6 +421,8 @@ def load(f, map_location=None, pickle_module=pickle, **pickle_load_args):
         new_fd = True
         f = f.open('rb')
     try:
+        if 'encoding' not in pickle_load_args.keys():
```

Comment:
@ezyang and @driazati Please advise if this way of specifying the default parameter is ok. I thought about adding `encoding` as a named parameter but that would require more than 10 changes to `unpickler.load` function call so this seemed like a more elegant solution.